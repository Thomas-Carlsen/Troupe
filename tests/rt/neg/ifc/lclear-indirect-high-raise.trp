(* negative ; receiving below the level of the pc at the raise time *)
let val _ = _setProcessDebuggingName ("main") 
in 
if 1 raisedTo {secret} then 
  let val c = raisembox ({secret})
   in rcv ({}, {secret}, [])
  end 
else 
  ()
end    