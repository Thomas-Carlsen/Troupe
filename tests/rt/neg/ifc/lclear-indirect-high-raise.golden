2020-02-15T11:53:01.340Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
[31mRuntime error in thread [main]0aa19a61-d3cc-44a7-a154-02527bf630a0@{}%{}[39m
[31m>> PC level at the time of raising the mailbox clearance is too sensitive for this receive[39m
[31m | receive lower bound: {}[39m
[31m | pc level at the time of receive: {secret}[39m
[31m | pc level at the time of raise: {secret}[39m
